distribution/test/itests/test-itests-ddf/src/test/java/ddf/test/itests/platform/TestSolrCommands.java

```diff
@@ -73,12 +74,14 @@ public void tearDown() {
     cleanUpBackups(CATALOG_CORE_NAME);
   }
 
+  @Ignore
```

Comment:
Why don't these work anymore?

Reply:
I thought these were using the http solr client, if I'm wrong I can re-enable these.

Reply:
So the solr commands don't work when using the embedded instance? If so we might want to make a not of that somewhere if we don't already.

Reply:
The backup command uses the HTTP endpoint, so it doesn't work when using the Embedded Solr. The documentation does mention that Embedded Solr must be backed up manually. I couldn't find any reference to the `solr:backup` command in the docs though. @ricklarsen?